elastic/kibana
# [RAM] Autocomplete

## Summary

Solves this issue: https://github.com/elastic/kibana/issues/161763

This PR introduces autocomplete for mustache variables for email connector(next PR will add it to all connectors) under the feature flag.

We decided keep old solution with button with all searchable options as well.

How to test:
Create an email connector in kibana.yml:

xpack.actions.preconfigured:
  maildev:
    name: 'email: maildev'
    actionTypeId: '.email'
    config:
      from: 'guskova@example.com'
      host: 'localhost'
      port: '1025'

How it should work:
You start writing in Message window {{ and mustache variable name. And you should see autocomplete popup with all possible options to choose. When you click somewhere else, popup should disappeared. 

https://github.com/elastic/kibana/assets/26089545/061016a6-b8ca-497b-9bed-b8b012d31a95

e options to choose. When you click somewhere else, popup should disappeared.
